### PR TITLE
Feature/v4 user endpoint php create

### PR DESCRIPTION
**Pull request by Rrathinasabapath**
Please review the changes for V4-User-endpoint-PHP-Create

### CODE REVIEW

File: tests/Hyperwallet/Tests/HyperwalletTest.php [hunk 1]
```diff
@@ -125,16 +124,35 @@ public function testCreateUser_withProgramTokenInUserObject() {
...
-        \Phake::when($apiClientMock)->doPost('/rest/v4/users', array(), $user, array())->thenReturn(array('success' => 'true'));
+        $expectedResponse = array('success' => 'true','verificationStatus'=>User::VERIFICATION_STATUS_VERIFIED,
+            'businessStakeholderVerificationStatus'=>User::BUSINESSS_STAKEHOLDER_VERIFICATION_STATUS_VERIFIED,
+            'letterOfAuthorizationStatus'=>User::LETTER_OF_AUTHORIZATION_STATUS_VERIFIED,
+            'governmentIdType'=>User::GOVERNMENT_ID_TYPE_NATIONAL_ID_CARD,
+            'firstName'=>"test-first-name",
+            'businessOperatingName'=>"test-business-operating-name");
+
+ //       \Phake::when($apiClientMock)->doPost('/rest/v4/users', array(), $user, array())->thenReturn(array('success' => 'true'));
+        \Phake::when($apiClientMock)->doPost('/rest/v4/users', array(), $user, array())->thenReturn($expectedResponse);
```
Comment (arao6) [inline tests/Hyperwallet/Tests/HyperwalletTest.php:142]: Please remove the commented line

File: src/Hyperwallet/Model/User.php [hunk 2]
```diff
@@ -46,7 +46,11 @@
  * @property string $language The user language
  * @property string $programToken The users program token
  * @property string $verificationStatus The status of user verification
+ * @property string $businessStakeholderVerificationStatus The status of Business Stakeholder verification
+ * @property string $letterOfAuthorizationStatus The status of Letter of Authorization verification
  * @property array $documents The array of documents returned for document upload
+ * @property string $governmentIdType The status of Letter of Authorization verification
+ * @property array $links The array of HATEOS links
  *
```
Comment (arao6) [inline src/Hyperwallet/Model/User.php:54]: The field 'timeZone' also needs to be added

File: tests/Hyperwallet/Tests/HyperwalletTest.php (hunk 1 above)
Comment (Rrathinasabapath) [reply tests/Hyperwallet/Tests/HyperwalletTest.php:142]: Done

File: src/Hyperwallet/Model/User.php (hunk 2 above)
Comment (Rrathinasabapath) [reply src/Hyperwallet/Model/User.php:54]: Done

**Review by arao6 — APPROVED** [review]

**Review by peter-joseph — APPROVED** [review]

**Pull request merged**